packages/calcite-components/src/components/combobox-item-group/combobox-item-group.tsx

```diff
@@ -53,6 +53,13 @@ export class ComboboxItemGroup extends LitElement {
    */
   @property() scale: Scale = "m";
 
+  /**
+   * Specifies whether the user set the hidden attribute in the HTML
+   *
+   * @private
+   */
+  @property({ reflect: true }) hiddenItem = false;
```

Comment:
Should we have e2e `defaults` and `reflects` tests for each of these new private properties?

Reply:
Not for internal props, but the controller approach could help us test this behavior.

Reply:
> Not for internal props, but the controller approach could help us test this behavior.

It would be beneficial even for internal props because if this property isn't reflected then the CSS and querySelector logic won't work. I would think it would be beneficial